data_collection/gazette/pipelines.py

```diff
@@ -167,4 +167,8 @@ def file_path(self, request, response=None, info=None, item=None):
         # class we replace that with the territory_id and gazette date.
         datestr = item["date"].strftime("%Y-%m-%d")
         filename = Path(filepath).name
+
+        if not filename.count(".pdf"):
```

Comment:
**problema** **Só** a extensão não indica se o arquivo é ou não é um PDF válido. Se queremos verificar isso em algum momento, é necessário analisar o conteúdo do arquivo. [Aqui](https://stackoverflow.com/questions/559096/check-whether-a-pdf-file-is-valid-with-python_) e [aqui](https://pypi.org/project/filetype/) temos algumas possibilidades. Porém é necessário analisar o impacto disso no processamento geral.

Outro problema é que existem cidades que fornecem o arquivo em outros formatos (como .docx). Sua solução vai adicionar erroneamente a extensão .pdf nesses arquivos.